bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -1605,6 +1618,26 @@ void DWARFRewriter::finalizeCompileUnits(DIEBuilder &DIEBlder,
                                          DIEStreamer &Streamer,
                                          CUOffsetMap &CUMap,
                                          const std::list<DWARFUnit *> &CUs) {
+  size_t BufferSize = 0;
+  for (DWARFUnit *CU : CUs) {
+    if (CU->getVersion() != 4)
+      continue;
+    std::optional<uint64_t> DWOId;
+    DWOId = CU->getDWOId();
+    if (DWOId) {
+      if (DIE *Die = UpdatedDIEsByDWO[*DWOId]) {
```

Comment:
What if DWOID is not in UpdatedDIEsByDWO? I believe I brought it up during draft of this.